package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -601,7 +603,18 @@ def _parse_box(self, header):
 
         return unitcell
 
-
+# Define the headers that define topology information from the lammps file
+# Any headers outside of this set will be ignored
+DUMP_HEADERS = set(
+    [
+        "id",
+        "mol",
+        "type",
+        "mass",
+        "element",
+        "q"
+    ]
+)
```

Comment:
I think much of the repetitive code can be consolidated by settings a dictionary like this. 
```suggestion
DUMP_HEADERS = {
    "id": {"atrr_class": Atomids, "default": None, "dtype": np.int32},
    "mol": {"atrr_class": [Resids, Resnums], "default": None, "dtype": int},
    "type": {"atrr_class": Atomtypes, "default": None, "dtype": object},
    "mass": {"atrr_class": Masses, "default": 1.0, "dtype": np.float64},
    "element": {"atrr_class": Atomids, "default": None, "dtype": object},
    "q": {"atrr_class": Charges, "default": None, "dtype": np.float32},
}
```